Gopkg.toml

```diff
@@ -100,37 +100,37 @@ required = [
   name = "github.com/spf13/viper"
   version = "1.0.0"
 
-[[constraint]]
+[[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.16.3"
+  version = "kubernetes-1.15.11"
```

Comment:
Shouldn't be necessary these are backwards compatible.

Reply:
Updated to newer 1.16.8 due to vulnerable dependencies in 1.16.3